addons/website_sale_stock/static/tests/tours/website_sale_stock_multilang.js

```diff
@@ -0,0 +1,48 @@
+/** @odoo-module **/
+
+import tour from 'web_tour.tour';
+
+tour.register('website_sale_stock_multilang', {
+    test: true,
+    url: '/fr/shop?search=unavailable',
+},
+[{
+    content: "Open unavailable product page",
+    trigger: 'a[content="unavailable_product"]',
+}, {
+    content: "Check out of stock message",
+    trigger: '#out_of_stock_message:contains("Hors-stock")',
+    run: () => {}, // This is a check.
+}, {
+    content: "Check price",
+    trigger: 'span:contains("123,45")',
+    run: () => {}, // This is a check.
+}, {
```

Comment:
Note that `extra_trigger` is generally a good fit to merge 2 check steps (same twice later)